Formula/ooniprobe.rb

```diff
@@ -200,41 +200,41 @@ def plist; <<-EOS.undent
    <dict>
      <key>Label</key>
        <string>#{plist_name}</string>
-     <key>Program</key>
-       <string>#{opt_bin}/ooniprobe</string>
+     <key>EnvironmentVariables</key>
+     <dict>
+       <key>PATH</key>
+       <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```

Comment:
It may be worth adding tor's `opt_bin` in here instead but I don't have strong feelings either way.